content/blog/what-happened-to-uuid-v2/index.md

```diff
@@ -18,13 +18,13 @@ If you read [my last post introducing UUIDs](/posts/what-are-uuids), you'll reme
 
 After this, [I left a very short (one or two sentences) explanation of what a UUIDv2 is](/posts/what-are-uuids#UUIDv2).
 
-Why would I do that [when UUIDv3 and UUIDv5 got four whole headers worth of explanation](/posts/what-are-uuids#UUIDv3and5)? What makes UUIDv2 different or "weird," as I put it?
+Why am I dedicating another article to UUIDv2? What makes it different or "weird?
```

Comment:
I'm not sure I like this edit. It changes the tone a bit too much for my liking:

```suggestion
Why would I do that [when UUIDv3 and UUIDv5 got four whole headers worth of explanation](/posts/what-are-uuids#UUIDv3and5)? What makes UUIDv2 different or "weird," as I put it?
```

Reply:
It was originally a suggestion since that sentence is very heavy, so you may reformulate it